proto/viam/service/slam/v1/slam.proto

```diff
@@ -26,6 +26,29 @@ service SLAMService {
       get: "/viam/api/v1/service/slam/{name}/map"
     };
   }
+
+  // GetPositionNew returns the current estimated position of the robot with
+  // respect to a returned reference frame
+  rpc GetPositionNew(GetPositionNewRequest) returns (GetPositionNewResponse) {
+    option (google.api.http) = {
+      get: "/viam/api/v1/service/slam/{name}/position_new"
+    };
+  }
+
+  // GetPointCloudMap returns the latest point cloud map generated by the
+  // SLAM library
+  rpc GetPointCloudMap(GetPointCloudMapRequest) returns (GetPointCloudMapResponse) {
+    option (google.api.http) = {
+      get: "/viam/api/v1/service/slam/{name}/point_cloud_map"
+    };
+  }
+  // GetInternalState returns the current map state as defined by the specified
```

Comment:
I would say "the internal state required to continue mapping/localizing".

Reply:
Changed to `GetInternalState returns the internal map as defined by the specified slam algorithm required to continue mapping/localizing` as I wanted to note that the returned internal state differences based on the algorithm. Let me know if you'd like me to remove the `... as defined by the specified slam algorithm`.

Reply:
can we make a note on this that it's not intended for end user use or something of the like?